ui/app/src/services/users.ts

```diff
@@ -157,3 +158,29 @@ export function fetchMyRolesBySite(sites?: Site[]): Observable<LookupTable<strin
     )
   );
 }
+
+export function fetchGlobalPreferences(): Observable<LookupTable<any>> {
+  return get('/studio/api/2/users/me/properties').pipe(pluck('response', 'properties', ''));
+}
+
+export function fetchSitePreferences(siteId: string): Observable<LookupTable<any>> {
+  return get(`/studio/api/2/users/me/properties?siteId=${siteId}`).pipe(pluck('response', 'properties'));
+}
+
+export function setPreferences(
+  properties: LookupTable<string>,
```

Comment:
LookupTable of string?

Reply:
should be any

Reply:
this is ok with string, because we should parse the object, boolean ,etc to string before send it to setPreferences

Reply:
close